devito/passes/iet/parpragma.py

```diff
@@ -213,9 +218,18 @@ def _make_reductions(self, partree):
         if not any(i.is_ParallelAtomic for i in partree.collapsed):
             return partree
 
+        def supported():
+            # Not all backend compilers support array reduction!
+            # Here are the known unsupported ones:
+            if isinstance(self.compiler, GNUCompiler) and \
```

Comment:
SHould we havd something similar for `osx clang + openmp`?

Reply:
ah, yeah, probably..